src/libs/ReportUtils.ts

```diff
@@ -5483,7 +5484,7 @@ function getReportSubtitlePrefix(report: OnyxEntry<Report>): string {
 /**
  * Get either the policyName or domainName the chat is tied to
  */
-function getChatRoomSubtitle(report: OnyxEntry<Report>, config: GetChatRoomSubtitleConfig = {isCreateExpenseFlow: false}): string | undefined {
+function getChatRoomSubtitle(report: OnyxEntry<Report>, config: GetChatRoomSubtitleConfig = {isCreateExpenseFlow: false, isReportArchived: false}): string | undefined {
```

Comment:
I think this config object is an anti-pattern, so please don't add any more properties to it and add `isReportArchived` as it's own parameter.

Of course, that leaves the config object with only a single property (hence, why it was an anti-pattern to begin with), so it would be nice for you to also move `isCreateExpenseFlow` to it's own parameter and change the name to something more appropriate like `preferPolicyName`. `isCreateExpenseFlow` makes this method too tightly coupled so it's difficult to reuse.